tui/src/state.rs

```diff
@@ -411,15 +411,23 @@ impl AppState {
         // This should be defined first to allow closing
         // the application even when not drawable ( If terminal is small )
         // Exit on 'q' or 'Ctrl-c' input
-        if matches!(
-            self.focus,
-            Focus::TabList | Focus::List | Focus::ConfirmationPrompt(_)
-        ) && (key.code == KeyCode::Char('q')
-            || key.modifiers.contains(KeyModifiers::CONTROL) && key.code == KeyCode::Char('c'))
+        if matches!(self.focus, Focus::TabList | Focus::List)
+            && (key.code == KeyCode::Char('q')
+                || key.modifiers.contains(KeyModifiers::CONTROL) && key.code == KeyCode::Char('c'))
         {
             return false;
         }
 
+        if matches!(self.focus, Focus::ConfirmationPrompt(_))
+            && (key.modifiers.contains(KeyModifiers::CONTROL) && key.code == KeyCode::Char('c'))
+        {
+            self.focus = Focus::List;
+            if !self.multi_select {
+                self.selected_commands.clear();
+            }
+            return true;
```

Comment:
![image](https://github.com/user-attachments/assets/d797b59a-bfdb-4e4f-ae70-c1152054103a)

No need for `Ctrl-C` to do the same thing. 

```suggestion
            return false;
```